.github/workflows/gateway-verify.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9']
+        python-version: ['3.9', '3.11']
```

Comment:
Similar to a previous comment, should we add here the rest of the python versions?